.wasser/config.yaml

```diff
@@ -55,10 +55,9 @@ routines:
         sudo systemctl start libvirtd
     - name: run microos image on vagrant
       command: |
-        cd aquarium/tests/vagrant
-        ./setup.sh foo 2>&1
-        cd setups/foo
-        vagrant up --no-destroy-on-error --no-parallel 2>&1
+        cd aquarium
+        ./tools/aqrdev create foo 2>&1
```

Comment:
Suggestion why not make 'aqrdev' to work without a need to change dir to  aquarium?

Reply:
at this point it requires to have the git repository somewhere as a parent, otherwise it will break. That's something I'm not entirely sure how to fix without having the user specifying the root of the repository.

Reply:
I usually calculate the root of repo from the __file__ path as a relation.

Reply:
yeah, that is one of the options, but I'm not so sure how that would play out if the file is moved in the future. Something to kick down the road and think about later on.

Reply:
well, when you move the file around, you tweak the relational path add or remove `..` :-D